scss/components/_blog.scss

```diff
@@ -0,0 +1,49 @@
+@import "headline";
+
+.blog-headline {
+  @extend %headline;
+}
+
+.blog-underline {
+  @extend %underline;
+  border-bottom: 1px solid #000;
+}
+
+.blog-underline-headline {
+  @extend %underline-headline;
+  color: #000;
+}
+
+.blog-underline-text {
+  @extend %underline-text;
+  align-self: center;
+  color: #000;
+  width: 40vw;
+}
+
+.test {
```

Comment:
you can add `:before, :after` inside the `.test`
```
.test { 
  &:before {}
  &:after {}
}
```

Reply:
Done